src/SearchIndexHandlers/Elasticsearch.php

```diff
@@ -58,6 +58,34 @@ public function upsertToIndex(Searchable $subject)
     }
 
     /**
+     * Add multiple searchable subjects to the index.
+     *
+     * @param array|Collection $subjects
+     */
+    public function insertManyToIndex($subjects = [])
+    {
+        $params = [];
+
+        foreach ($subjects as $subject) {
+            if (! $subject instanceof Searchable) {
+                throw new \InvalidArgumentException;
+            }
+
+            $params['body'][] = [
+                "index" => [
+                    '_id' => $subject->getSearchableId(),
```

Comment:
Yes, it is necessary according to documentation https://www.elastic.co/guide/en/elasticsearch/client/php-api/current/_indexing_documents.html#_bulk_indexing